Raven.Studio.Html5/App/views/manage/cluster.html

```diff
@@ -47,6 +47,8 @@ <h3 class="text-info">
                             <li><a href="#" data-bind="click: initializeNewCluster">Secede from cluster - rude step down into a single node cluster</a></li>
                             <li><a href="#" data-bind="click: addAnotherServerToCluster.bind($root, true)">Kidnap node into cluster - force a node to the current cluster</a></li>
                             <li><a href="#" data-bind="click: removeClustering.bind($root)">Remove cluster entirely and perform cleanup.</a></li>
+                            <li data-bind="visible: !isReplicationCheckEnabled()"><a href="#" data-bind="click: toggleReplicationChecks.bind($root)">Disable replication state sync.</a></li>
+                            <li data-bind="visible: isReplicationCheckEnabled()"><a href="#" data-bind="click: toggleReplicationChecks.bind($root)">Prevent out of date leaders.</a></li>
```

Comment:
UI changes like that are your expertise.

*Hibernating Rhinos Ltd  *

Oren Eini* l CEO l *Mobile: + 972-52-548-6969

Office: +972-4-622-7811 *l *Fax: +972-153-4-622-7811



On Wed, Aug 17, 2016 at 10:47 PM, Marcin Lewandowski <
notifications@github.com> wrote:

> In Raven.Studio.Html5/App/views/manage/cluster.html
> <https://github.com/ayende/ravendb/pull/4238#discussion_r75192720>:
>
> > @@ -47,6 +47,8 @@ <h3 class="text-info">
> >                              <li><a href="#" data-bind="click: initializeNewCluster">Secede from cluster - rude step down into a single node cluster</a></li>
> >                              <li><a href="#" data-bind="click: addAnotherServerToCluster.bind($root, true)">Kidnap node into cluster - force a node to the current cluster</a></li>
> >                              <li><a href="#" data-bind="click: removeClustering.bind($root)">Remove cluster entirely and perform cleanup.</a></li>
> > +                            <li data-bind="visible: !isReplicationCheckEnabled()"><a href="#" data-bind="click: toggleReplicationChecks.bind($root)">Disable replication state sync.</a></li>
> > +                            <li data-bind="visible: isReplicationCheckEnabled()"><a href="#" data-bind="click: toggleReplicationChecks.bind($root)">Prevent out of date leaders.</a></li>
>
> Can we replace this with data-bind="text: isReplicationCheckEnabled()?
> 'Prevent out of date leaders.':'Disable replication state sync' ?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ayende/ravendb/pull/4238/files/d2e118adfd5d99d145660a0f1350bc38cec6bf18#r75192720>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AB4wANWQzX6MFh2k6JL3S78hNo72IRTyks5qg2VXgaJpZM4JmXlq>
> .
>